ChangeLog

```diff
@@ -1,5 +1,9 @@
  # ChangeLog for Pod::Simple dist
 #---------------------------------------------------------------------------
+3.43 2021-06-15 Karl Williamson <khw@cpan.org>
```

Comment:
3.43 hasn't been released yet, so this date is wrong?  I'd just omit the date until it's time to release.